Software/Backend/DataAccessLayer/Infrastructure/JobRepository.cs

```diff
@@ -11,7 +12,8 @@ namespace DataAccessLayer.Infrastructure
     public class JobRepository : IJobRepository
     {
         private readonly DB _db;
-        public JobRepository(DB db) {
+        public JobRepository(DB db)
+        {
```

Comment:
Ovakve stvari ne moraš commitati, kad se nešto tako refaktorira, može samo izazvati konflikte.